### PR TITLE
Tests - Change structure of file globals.js

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Change structure of file globals.js, grouping vars by utility 
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | URL_FO=SHOP_URL npm run sanity-tests

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15754)

### CODE REVIEW

File: tests/puppeteer/campaigns/utils/globals.js [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+global.INSTALL = {
+  URL: process.env.URL_INSTALL || `${global.FO.URL}install-dev/`,
+  LANGUAGE: process.env.INSTALL_LANGUAGE || 'en',
+  COUNTRY: process.env.INSTALL_COUNTRY || 'fr',
+  DB_USER: process.env.DB_USER || 'root',
+  DB_PASSWD: process.env.DB_PASSWD || '',
+  SHOPNAME: process.env.SHOPNAME || 'Prestashop',
+  PS_VERSION: process.env.PS_VERSION || '1.7.6.0',
+};
+global.browserConfig = {
```
Comment (SimonGrn) [inline tests/puppeteer/campaigns/utils/globals.js]: No caps ?

**Review by SimonGrn — DISMISSED** [review]
Thanks @boubkerbribri

File: tests/puppeteer/campaigns/utils/globals.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/campaigns/utils/globals.js]: I change it to `BROWSER_CONFIG` but for attributes inside I can't; Puppeteer will not accept caps.

**Review by matthieu-rolland — APPROVED** [review]
It's nice to namespace things like this, thank you @boubkerbribri !

**Pull request merged**